src/components/Panel.vue

```diff
@@ -4,16 +4,22 @@
       v-if="$slots.header"
       :class="suit('header')"
     >
-      <slot name="header" />
+      <slot
+        name="header"
+        no-refinement="!canRefine"
```

Comment:
I think the proper English way is "no refinements". But this is a mistake that we made in InstantSearch.css.